PowerEditor/src/Notepad_plus.cpp

```diff
@@ -2625,7 +2596,9 @@ void Notepad_plus::maintainIndentation(TCHAR ch)
 			int startPos = static_cast<int32_t>(_pEditView->execute(SCI_GETCURRENTPOS));
 			if (startPos != 0)
 				startPos -= 1;
-			int posFound = findMachedBracePos(startPos - 1, 0, '{', '}');
+
+			_pEditView->execute(SCI_COLOURISE, 0, -1);

```

Comment:
I don't think calling `SCI_COLOURISE` is a good idea because it forces the entire document to get re-lexed.

Reply:
Maybe added because of https://scintilla-interest.narkive.com/TWBDw58B/problem-with-sci-bracematch